keras_imaging/preprocessing/standardize.py

```diff
@@ -73,3 +58,29 @@ def f(x):
         return y
 
     return f
+
+
+def normalization(means, max_values):
+    """
+    normalization make the image to range [-1,1]
+    :param means: means of image values across the dataset
+    :param max_values: maximum value
+    :return: normalized image
+    """
+    def f(x):
+        return (x - means) / max_values
+
+    return f
+
+
+def gaussian_blur(mean_sigma, variance_sigma):
```

Comment:
I don’t think it’s necessary to specify variance. We’re really trying to emphasize convention over configuration.